config/control-plane/kustomization.yaml

```diff
@@ -18,7 +18,6 @@ components:
   - ../watcher
   - ../webhook
   - ../grafana
-  - ../certmanager
```

Comment:
maybe because I'm not following the whole issue, why we decide to remove certmanager here? We added it in the management control plane later?